ecl/eclcc/eclcc.cpp

```diff
@@ -2063,6 +2068,10 @@ bool EclCC::parseCommandLineOptions(int argc, const char* argv[])
         {
             setDebugOption("syntaxCheck", tempBool);
         }
+        else if (iter.matchFlag(tempBool, "--auto-id-suggestions"))
```

Comment:
Yep, thanks,  I forgot to change this, it was left over from when I had the internal bool option named optNoAutoIdSugestion and thus had to invert.